ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -140,7 +135,6 @@ Item {
             height: parent.height
             visible: !root.rootStore.openCreateChat && isActiveChannel
             chatId: model.itemId
-            activeChatType: root.activeChatType
```

Comment:
I think it's probably still faster/better to get it from the model

Reply:
Done